container/image.bzl

```diff
@@ -238,7 +238,7 @@ def _impl(ctx, files=None, file_map=None, empty_files=None, directory=None,
 
   file_map = file_map or {}
   files = files or ctx.files.files
-  empty_files = empty_files or ctx.attr.empty_files
+  empty_files = empty_files or {f: f for f in ctx.attr.empty_files}
```

Comment:
Why do we need to change this from a list to dictionary? The only uses seem to be in calling `_build_layer`, which isn't changed in this PR so I imagine it doesn't care, and in creating `transient_emptyfiles`, which could be done with the list directly. Am I missing something?
Also, if this change is indeed necessary, we should probably update the documentation of this function to reflect the new type.

Reply:
I used to pass relative paths in the dict value. But now I reshuffled things and found out it's not needed, so I removed this confusing thing.